### PR TITLE
#trivial Update docs

### PR DESCRIPTION
**Pull request by kevinrodrigues**
Docs for consuming applications.

### CODE REVIEW

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+If you are upgrading from an older version of fozzie – i.e v8 and older – then this migration guide will help you update to the latest version (v9) which uses `dart-sass` as opposed to `node-sass`
+
+<a name="scheme"></a><h2>Prerequisites</h2>
+
+Before you get started with the Fozzie SCSS helper library there are a few things you need to know in order to get up and running.
+
+From v9 and above, you'll need the following:
+
+1. [dart-sass](https://sass-lang.com/dart-sass):
+    - Fozzie makes use of new SCSS syntax to compile your Sass (namely `@use` & `@forward`). The [sass](https://www.npmjs.com/package/sass) module now uses dart-sass by default, so if you use the latest version of this module, you'll be fine.
```
Comment (FayeCarter) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx]:
```suggestion
    - Fozzie makes use of new SCSS syntax to compile your Sass (namely `@use` & `@forward`). The [sass](https://www.npmjs.com/package/sass) module now uses `dart-sass` by default, so if you use the latest version of this module, you'll be fine.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+### Common.scss
+
+```
+┣ assets
+    ┣ scss
+        ┣ common.scss
+```
+
+You may have a file like `Common.scss` that will usually contain all of the shared styles required throughout an application. If this is the case you should have a setup like below which effectively imports `fozzie` and other helper files making them globally available throughout the application.
```
Comment (FayeCarter) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx]:
```suggestion
You may have a file like `common.scss` that will usually contain all of the shared styles required throughout an application. If this is the case you should have a setup like below which effectively imports `fozzie` and other helper files making them globally available throughout the application.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+@include normalize();
+@include reset();
+@include typography();
+@include links();
+@include trumps-utilities();
+```
+
+#### New way (@use / @forward):
+
+With the new way to import or "use / forward" scss files we can change the above to the following:
```
Comment (FayeCarter) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx]:
```suggestion
With the new way to import or `@use`/`@forward` scss files we can change the above to the following:
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+@use '~@justeat/fozzie/src/scss/fozzie' as f;
+
+@include f.normalize();
+@include f.reset();
+@include f.typography();
+@include f.links();
+@include f.trumps-utilities();
+```
+
+`Note` the namespacing references, there's more on how namespacing works [here](https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace). If you have scss files like a header and footer shared across your application stored in your common.scss directory then you would simply @forward these on to be used within the Vue single file component like so:
```
Comment (FayeCarter) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx]:
```suggestion
`Note` the namespacing references, there's more on how namespacing works [here](https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace). If you have scss files like a header and footer shared across your application stored in your common.scss directory then you would simply `@forward` these on to be used within the Vue single file component like so:
```

**Review by jamieomaguire — DISMISSED** [review]

**Review by FayeCarter — DISMISSED** [review]

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,145 @@
+<Meta title="Documentation/Guides/Fozzie/Fozzie Migration Guide"/>
+
+# Fozzie version 9 Migration guide for consuming applications
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:3]:
```suggestion
# Fozzie version 9 Migration guide
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+If you are upgrading from an older version of fozzie – i.e v8 and older – then this migration guide will help you update to the latest version (v9) which uses `dart-sass` as opposed to `node-sass`
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:5]:
```suggestion
If you are upgrading from an older version of fozzie – i.e v8 and older – then this migration guide will help you update to the latest version (v9) which uses `dart-sass` as opposed to `node-sass`.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+<a name="scheme"></a><h2>Prerequisites</h2>
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:7]:
```suggestion
<a name="prerequisites"></a><h2>Prerequisites</h2>
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+Before you get started with the Fozzie SCSS helper library there are a few things you need to know in order to get up and running.
+
+From v9 and above, you'll need the following:
+
+1. [dart-sass](https://sass-lang.com/dart-sass):
+    - Fozzie makes use of new SCSS syntax to compile your Sass (namely `@use` & `@forward`). The [sass](https://www.npmjs.com/package/sass) module now uses `dart-sass` by default, so if you use the latest version of this module, you'll be fine.
+2. [webpack](https://webpack.js.org/) (or a similar file bundler)
+    - For file resolution paths into the `node_modules` folder. If you aren't using Webpack, then you can use something like [ParcelJs](https://parceljs.org/) instead, which also supports this.
+
+<a name="scheme"></a><h2>Removing node-sass in favour of sass</h2>
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:18]:
```suggestion
<a name="removing-node-sass"></a><h2>Removing node-sass in favour of sass</h2>
```

**Review by jamieomaguire — DISMISSED** [review]

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+
+If you’re a user of Node Sass, migrating to Dart Sass is straightforward: just replace `node-sass` in your package.json file with `sass`. Both packages expose the same JavaScript API.
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:20]:
```suggestion
If you use Node Sass, migrating to Dart Sass is straightforward; just replace `node-sass` in your `package.json` file with `sass`. Both packages expose the same JavaScript API.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+  "name": "Example",
+  "version": "0.0.1",
+  "description": "Example",
+  "devDependencies": {
+    "sass": "1.52.3", << node-sass changed for `sass`
+  }
+}
+```
+
+If you have an old version of `fozzie` installed within your application at this point, you will likely see some syntax errors. In order to resolve these issue you will need to move to `v9` of `fozzie` which currently sits in [beta](https://www.npmjs.com/package/@justeat/fozzie/v/9.0.0-beta.5).
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:48]:
```suggestion
If you have an old version of `fozzie` included within your application at this point, you will likely see some syntax errors. In order to resolve these issues you will need to move to `v9` of `fozzie`, which currently sits in [beta](https://www.npmjs.com/package/@justeat/fozzie/v/9.0.0-beta.5).
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+This version of `fozzie` contains changes which will automatically resolve those errors.
+
+<a name="scheme"></a><h2>How to move from `@import` to `@use` & `@forward`</h2>
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:51]:
```suggestion
<a name="use-and-forward"></a><h2>How to move from `@import` to `@use` & `@forward`</h2>
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+
+Once you have moved over to `dart-sass` and switched to the new version of fozzie (v9). You can begin to update your application with the new `@use` & `@forward` syntax. If you are unfamiliar with how `@use` & `@forward` work there's a useful video found [here](https://www.youtube.com/watch?v=CR-a8upNjJ0&ab_channel=KevinPowell) worth watching first.
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:53]:
```suggestion
Once you have moved over to `dart-sass` and switched to the new version of fozzie (v9). You can begin to update your application with the new `@use` & `@forward` syntax. If you are unfamiliar with how `@use` & `@forward` work, we'd recommend [watching this video](https://www.youtube.com/watch?v=CR-a8upNjJ0&ab_channel=KevinPowell) first.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+
+### Common.scss
+
+```
+┣ assets
+    ┣ scss
+        ┣ common.scss
+```
+
+You may have a file like `common.scss` that will usually contain all of the shared styles required throughout an application. If this is the case you should have a setup like below which effectively imports `fozzie` and other helper files making them globally available throughout the application.
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:74]:
```suggestion
You may have a file like `common.scss` that contains all of the shared styles required throughout an application. If this is the case, you should have a setup like below which effectively imports `fozzie` and other helper files, making them globally available throughout the application.
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+@include normalize();
+@include reset();
+@include typography();
+@include links();
+@include trumps-utilities();
+```
+
+#### New way (@use / @forward):
+
+With the new way to import or `@use`/`@forward` scss files we can change the above to the following:
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:89]:
```suggestion
Using the new `@use`/`@forward` syntax, we can change the above to the following:
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+@use '~@justeat/fozzie/src/scss/fozzie' as f;
+
+@include f.normalize();
+@include f.reset();
+@include f.typography();
+@include f.links();
+@include f.trumps-utilities();
+```
+
+`Note` the namespacing references, there's more on how namespacing works [here](https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace). If you have scss files like a header and footer shared across your application stored in your common.scss directory then you would simply `@forward` these on to be used within the Vue single file component like so:
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:101]:
```suggestion
`Note` the namespacing references, you can read more [on how namespacing works here](https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace).

If you have scss files like a header and footer shared across your application stored in your `common.scss` directory, then you would simply `@forward` these on to be used within the Vue single file component like so:
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+// Single File Component (SFC)
+// default.vue
+
+<style lang="scss">
+@use '../assets/scss/common';
+</style>
+
+```
+
+<a name="scheme"></a><h2>Vue config</h2>
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:119]:
```suggestion
<a name="vue-config"></a><h2>Vue config</h2>
```

File: stories/guides/fozzie/fozzie-migration-guide.stories.mdx [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+
+Your application may load in a common.scss file at `vue.config` level (see below) & [here](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-header/vue.config.js#L21), if that's the case you can still make use of `@use`:
```
Comment (ashleynolan) [inline stories/guides/fozzie/fozzie-migration-guide.stories.mdx:121]:
```suggestion
Your application may load in a `common.scss` file at the `vue.config` level (see below) & [here](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-header/vue.config.js#L21). If that's the case you can still make use of `@use`:
```

**Review by ashleynolan — COMMENTED** [review]
Some super minor comments. Looks good 👍

**Pull request closed without merging**